README.md

```diff
@@ -2,13 +2,69 @@
 
 [![CircleCI](https://circleci.com/gh/Khan/flow-to-ts.svg?style=svg)](https://circleci.com/gh/Khan/flow-to-ts) [![codecov](https://codecov.io/gh/Khan/flow-to-ts/branch/master/graph/badge.svg)](https://codecov.io/gh/Khan/flow-to-ts)
 
-Convert flow code to typescript.
+Convert Flow code to TypeScript.
+
+The goal of thiis project is to provide a tool that can translate 95% of Flow 
+to TypeScript while maintaining a high percentage of the existing type 
+information.  We don't want to convert code and end up with everything using
+`any`.  We also want to avoid having to make a lot of manual changes to files
+afterwards, e.g. changing `SyntheticEvent` to `React.Event`.
 
 # Quick start
 
-- yarn
-- yarn test
+- yarn global add @khanacademy/flow-to-ts
+- flow-to-ts [options] <file globs>
 
 # Playground
 
 https://flow-to-ts.netlify.com
+
+# Principles
+
+- when exact translation isn't possible:
+  - downgrade to `any` when possible and provide a gentle warning
+  - when it isn't possible to downgrade (e.g. `%checks`), remove the syntax
+    and provide a forceful warning that the code in question will need a human
+    to convert it manually.
+- best effort to maintain blank lines and the position comments, this isn't
+  always possible so warn in those situations where we can't, e.g. converting
+  object type spreads to intersection types.
+
+# Roadmap
```

Comment:
I think it might be worth keeping the roadmap in an external issue, that way it's easier to update and track - that way you don't need to make a PR every time you want to update it!